rotkehlchen/errors.py

```diff
@@ -90,10 +93,19 @@ class DeserializationError(Exception):
 
 
 class NoPriceForGivenTimestamp(Exception):
-    def __init__(self, from_asset: 'Asset', to_asset: 'Asset', date: str) -> None:
+    def __init__(self, from_asset: 'Asset', to_asset: 'Asset', date: 'Timestamp') -> None:
+        self.from_asset = from_asset
+        self.to_asset = to_asset
+        self.date = date
         super().__init__(
             'Unable to query a historical price for "{}" to "{}" at {}'.format(
-                from_asset.identifier, to_asset.identifier, date,
+                from_asset.identifier,
+                to_asset.identifier,
+                timestamp_to_date(
+                    date,
```

Comment:
also here. And remember to always use kwargs

```suggestion
                    ts=time,
```